generator/src/lib.rs

```diff
@@ -2755,6 +2888,9 @@ pub fn write_source_code<P: AsRef<Path>>(vk_xml: &Path, src_dir: P) {
         pub use feature_extensions::*;
         mod features;
         pub use features::*;
+        #[allow(nonstandard_style)]
+        mod native;
+        pub use native::*;
```

Comment:
Haven't really thought whether we should reexport native types like this, or "hide" them behind `pub mod native` instead.

Reply:
I think this should not be re-exported here, and just a `pub mod native` as the types in there are pretty different. Could also drop a doc comment above it to indicate that all of those types were generated by bindgen.

i think you can maybe do a `pub(crate) use native::*;` here

Reply:
The glob is already used in `use crate::vk::native::*;` in `definitions.rs`, which should not publicly reexport them (ie. they are not caught by `pub use definitions::*;`), so it's fine to drop the `* use native::*;` here entirely.